src/components/annotations/variants/VariantsView.vue

```diff
@@ -61,10 +61,12 @@ onBeforeUnmount(() => unsubscribe())
 
 function allocateWitnessColorInVariantItem() {
   const colors = {}
-  annotationStore.witnesses.forEach((witness, i) => {
+  if(annotationStore.witnesses?.length > 0) {
```

Comment:
Please use "early return" pattern, since wrapping an "if" around some code is kind of ugly